src/components/Header/LayerRoute/LayerRoute.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react';
+
+import useMap from '@hook/useMap';
+import { useSelectedState } from '@store/state/selectedContext';
+import { useSelectedDistrict } from '@store/district/selectedContext';
+
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+
+import * as Styles from './styles';
+
+const LayerRoute = () => {
+  const [selectedLayer, setSelectedLayer] = useState('country');
+
+  const { resetMapValues, resetDistrictValues } = useMap();
+  const { selected: selectedState } = useSelectedState();
+  const { selected } = useSelectedDistrict();
+
+  useEffect(() => {
+    if (selected) {
+      setSelectedLayer('district');
+    } else if (selectedState && !selected) {
+      setSelectedLayer('state');
+    } else {
+      setSelectedLayer('country');
+    }
+  }, [selected, selectedState]);
+
+  const returnPath = () => {
+    if (selectedLayer === 'district') {
+      return (
+        <>
+          &nbsp;-&nbsp;
+          <div className="state" onClick={() => resetDistrictValues()}>
+            {selectedState?.properties.NM_UF}
+          </div>
+          &nbsp;-&nbsp;<div className="district">{selected?.properties.NM_MUN}</div>
+        </>
```

Comment:
Idk if it was intentional this div being on the same line.

Reply:
Fixed!